scripts/restore

```diff
@@ -38,15 +38,29 @@ NGINX_CONF="/etc/nginx/conf.d/${domain}.d/${app}.conf"
 ynh_restore_file "$NGINX_CONF"
 
 #=================================================
-# CONFIGURE SSOWAT
+# SETUP SSOWAT
 #=================================================
 
+if [[ "$is_public" -eq 0 ]]
+then	# Remove the public access
+	ynh_app_setting_delete "$app" skipped_uris
+fi
```

Comment:
Eeeh there's no is_public define in this script though ? And it shouldn't exist anyway ?